geode-core/src/main/java/org/apache/geode/internal/cache/AbstractBucketRegionQueue.java

```diff
@@ -257,11 +273,45 @@ protected void loadEventsFromTempQueue() {
             }
             getInitializationLock().writeLock().unlock();
           }
+          if (regionToDuplicateEventsMap.size() > 0
```

Comment:
Can this new phase of of this method be extracted into a new method?

Reply:
Updated.